build.gradle

```diff
@@ -75,6 +75,8 @@ dependencies {
             transitive false
         }
     }
+compileOnly 'com.github.SoKnight.PEconomy:PEconomy:1.2.0'
+}
```

Comment:
This should be in the dependency list in `core/build.gradle`, the other submodules don't need to have this dependency. Also looks like there's an extra bracket there?